### PR TITLE
Add rain token

### PR DESCRIPTION
**Pull request by ivardl**
adding Rain token

### CODE REVIEW

File: tokens/main/0x3Cb62f48DBdC4f7627f37F027811565d292a1001/additional.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "website":"earth",
```
Comment (XJWX89) [inline tokens/main/0x3Cb62f48DBdC4f7627f37F027811565d292a1001/additional.json:2]: We do not currently accept this format.

File: tokens/test/0x3Cb62f48DBdC4f7627f37F027811565d292a1001/additional.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "website":"earth",
```
Comment (XJWX89) [inline tokens/test/0x3Cb62f48DBdC4f7627f37F027811565d292a1001/additional.json:2]: We do not currently accept this format.

**Review by XJWX89 — CHANGES_REQUESTED** [review]
hi @ivardl , Contract address start with 0x + 40 characters and **must be lower case**

File: tokens/main/0x3Cb62f48DBdC4f7627f37F027811565d292a1001/additional.json (hunk 1 above)
Comment (ivardl) [reply tokens/main/0x3Cb62f48DBdC4f7627f37F027811565d292a1001/additional.json]: have updated with an actual website.

**Comment by ivardl** [comment]
I made my folders all lowercase before the new commit but Im not sure this carried over. Please advise if I still did something wrong with the contract address structure.

**Comment by ivardl** [comment]
@XJWX89 i did a commit deleting the old folders and another commit with new folders. Hopefully this resolves the folder name issue. Thank you.

**Review by libotony — APPROVED** [review]

**Review by XJWX89 — COMMENTED** [review]
hi @ivardl , sorry for the late reply. It seems that the contract address on mainnet and testnet are the same. if you do not have a contract on testnet ,You can simply provide the mainnet contract address.

**Comment by ivardl** [comment]
Thanks for your patience. It looks like i may have changed more files than I intended. Please let me know if i broke anything. I think i deleted the testnet folder for rain. thank you.

**Comment by libotony** [comment]
> Thanks for your patience. It looks like i may have changed more files than I intended. Please let me know if i broke anything. I think i deleted the testnet folder for rain. thank you.

Please do not delete other token's folder.

**Pull request closed without merging**